File: Python/Bioskop.py
class Bioskop:
    # Constructor
    def __init__(self, id: str = "", judul: str = "", genre: str = "", durasi: int = 0, foto: str = ""):
        self.__id = str(id)
        self.__judul = str(judul)
        self.__genre = str(genre)
        self.__durasi = int(durasi)
        self.__foto = str(foto)

    # Getter & Setter ID
    def getId(self) -> str:
        return self.__id

    def setId(self, id: str) -> None:
        self.__id = str(id)

    # Getter & Setter Judul
    def getJudul(self) -> str:
        return self.__judul

    def setJudul(self, judul: str) -> None:
        self.__judul = str(judul)

    # Getter & Setter Genre
    def getGenre(self) -> str:
        return self.__genre

    def setGenre(self, genre: str) -> None:
        self.__genre = str(genre)

    # Getter & Setter Durasi
    def getDurasi(self) -> int:
        return self.__durasi

    def setDurasi(self, durasi: int) -> None:
        self.__durasi = int(durasi)

    # Getter & Setter Foto
    def getFoto(self) -> str:
        return self.__foto

    def setFoto(self, foto: str) -> None:
        self.__foto = str(foto)

File: Java/Bioskop.java
public class Bioskop {
    // Atribut privat (Encapsulation)
    private String id;
    private String judul;
    private String genre;
    private int durasi;
    private String foto;

    // Constructor kosong
    public Bioskop() {
        this.id = "";
        this.judul = "";
        this.genre = "";
        this.durasi = 0;
        this.foto = "";
    }

    // Constructor berparameter
    public Bioskop(String id, String judul, String genre, int durasi, String foto) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.durasi = durasi;
        this.foto = foto;
    }

    // Getter dan Setter
    public String getId() {
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getJudul() {
        return this.judul;
    }

    public void setJudul(String judul) {
        this.judul = judul;
    }

    public String getGenre() {
        return this.genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public int getDurasi() {
        return this.durasi;
    }

    public void setDurasi(int durasi) {
        this.durasi = durasi;
    }

    public String getFoto() {
        return this.foto;
    }

    public void setFoto(String foto) {
        this.foto = foto;
    }
}
